src/main/java/tempest/ui/GUIManager.java

```diff
@@ -110,4 +127,8 @@ public Page getPage(Class<? extends Page> classObject) {
     public String getCurrentCard() {
         return vm.getVisible();
     }
+
+    public void closeGUI() {
```

Comment:
JavaDoc this, please.

Reply:
I've added JavaDoc - the method isn't used by anything at the moment though.

We might want to use it if later we make tests to show that upon reopening the GUI the changes to modules and sessions have stayed.